### DISCUSSION TITLE
`delay` prop replacement in `<Tooltip>` when migrating to v3

### DISCUSSION THREAD

**Discussion by anmolshres98** [iTwin/iTwinUI#2352] [Q&A] [answered]
Hi, I am migrating my app to use `itwinui-react` v3 form v2 and it appears that the `delay` prop is no longer supported in the `<Tooltip>` component. I couldn't discern the alternative when looking at the migration guide. 

How can I pass in delay value for the new tooltip component?

**Comment by mayank99** [comment]
Can you please share your use-case for the `delay` prop? It isn't exposed because you shouldn't ever need to override the default delay.

**Reply by anmolshres98** [reply]
I see, I am just trying to migrate the code one-to-one so I thought I needed it since the current code while using v2 is passing it in. Does this mean the tooltip comes with a default override? if yes, what value is that?

**Reply by mayank99** [reply, marked as answer]
Yes, the `Tooltip` currently has a delay of `50` ms (for open) + `250` ms (for close). These values might change in the future after UX discussions.
